test/integration/verify/controls/aws_amis.rb

```diff
@@ -0,0 +1,28 @@
+title 'Test AWS AMIs in bulk'
+
+control 'aws-amis-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS AMI has current properties'
+
+  describe aws_amis(image_id: 'ami-0a13d44dccf1f5cf6', architecture: 'x86_64') do
```

Comment:
Instead of hard-coding the AMI image ID (which will eventually go stale and cause CI to break) can we rework this control as a plural->singular resource example?  The aws_amis resource will have to be suitably filtered to reduce the number of results as much as possible and the individual tests reworked according to the properties of the result set.

Reply:
Good call, latest commit updated tests to be more dynamic.